app/javascript/components/journey/ContributionsList.tsx

```diff
@@ -45,12 +46,17 @@ export function ContributionsList({
     request.endpoint,
     removeEmpty(request.query),
   ]
-  const { status, resolvedData, latestData, isFetching, error } =
-    usePaginatedRequestQuery<APIResult>(cacheKey, {
-      ...request,
-      query: removeEmpty(request.query),
-      options: { ...request.options, enabled: isEnabled },
-    })
+  const {
+    status,
+    data: resolvedData,
+    isFetching,
+    error,
+  } = usePaginatedRequestQuery<APIResult>(cacheKey, {
+    ...request,
+    query: removeEmpty(request.query),
+    options: { ...request.options, enabled: isEnabled },
+  })
```

Comment:
I've seen this pattern a couple of times, where empty values are removed and `enabled` is set. It could be useful to, at some point, extract this to a helper.